src/layers/conv.jl

```diff
@@ -161,7 +161,8 @@ end
 @functor Conv
 
 function (c::Conv)(x::AbstractArray)
-  σ, b = c.σ, reshape(c.bias, ntuple(_ -> 1, length(c.stride))..., :, 1)
+  b = reshape(c.bias, ntuple(_ -> 1, length(c.stride))..., :, 1)
```

Comment:
This line could be pulled into a helper function. I also noticed that `Conv` uses `ntuple` while the others use `map`.

Reply:
Could be. Maybe revisit with #1832?

Reply:
How about an internal only function for now? Or barring that, updating this line to match.
```suggestion
  b = reshape(c.bias, map(_->1, c.stride)..., :, 1)
```